SKILL.tsv

```diff
@@ -3750,38 +3750,38 @@ SKILL_20150414_003749	Use pluvial of alchemy and create a new item.
 SKILL_20150414_003751	{memo X}Each weapon has different dungeon{nl}and potential effects{nl}Level 1 Master
 SKILL_20150414_003752	{memo X}{#DD5500}{ol}[Physical] - [Pierce]{/}{/}{nl}Quickly shoot multiple arrows.
-SKILL_20150414_003753	{#DD5500}{ol}[Physical] - [Pierce]{/}{/}{nl}Pull bow and shoot an arrow that penetrates the enemy. Target may be skewed if there are other enemies nearby.
+SKILL_20150414_003753	{#DD5500}{ol}[Physical] - [Pierce]{/}{/}{nl}Draw your bowstring and fire an arrow that penetrates enemies. Enemies close to one another may be skewered together.
 SKILL_20150414_003754	{memo X}{#DD5500}{ol}[Physical] - [Pierce]{/}{/}{nl}Sit down to increase focus. ATK and range increases.
 SKILL_20150414_003755	$Temporarily increases your movement speed while attacking.
-SKILL_20150414_003756	{#DD5500}{ol}[Physical] - [Pierce]{/}{/}{nl}Arrow bounces off attacking one more enemy.
-SKILL_20150414_003757	$Heavy Shot
-SKILL_20150414_003758	{#DD5500}{ol}[Physical] - [Pierce]{/}{/}{nl}Pull bow strongly to shoot and target is pushed away.
-SKILL_20150414_003759	$Twins Arrow
+SKILL_20150414_003756	{#DD5500}{ol}[Physical] - [Pierce]{/}{/}{nl}Arrow ricochets to a nearby enemy.
+SKILL_20150414_003757	$HeavyShot
+SKILL_20150414_003758	{#DD5500}{ol}[Physical] - [Pierce]{/}{/}{nl}Demonstrate your strength by firing an arrow that pushes enemies away.
+SKILL_20150414_003759	$TwinArrows
 SKILL_20150414_003760	{#DD5500}{ol}[Physical] - [Pierce]{/}{/}{nl}Shoot two arrows at once.
-SKILL_20150414_003761	{#DD5500}{ol}[Physical] - [Pierce]{/}{/}{nl}Use tip of arrow to aim at target and penetrate stright forward.
-SKILL_20150414_003762	{#DD5500}{ol}[Physical] - [Pierce]{/}{/}{nl}Shoot arrow with high rate of critical.
+SKILL_20150414_003761	{#DD5500}{ol}[Physical] - [Pierce]{/}{/}{nl}Penetrate straight through the enemy by aiming with the tip of your arrow. 
+SKILL_20150414_003762	{#DD5500}{ol}[Physical] - [Pierce]{/}{/}{nl}Shoot an arrow with a high critical chance.
 SKILL_20150414_003763	{memo X}Attack speed decrease and ATK increase.
-SKILL_20150414_003764	{#DD5500}{ol}[Physical] - [Strike]{/}{/}{nl}Fire arrows with bomb. Arrows explode after hitting enemy.
+SKILL_20150414_003764	{#DD5500}{ol}[Physical] - [Strike]{/}{/}{nl}Fire an arrow with a bomb attached. Arrows explode after a short amount of time. 
 SKILL_20150414_003765	{memo X}{#DD5500}{ol}[Physical] - [Pierce]{/}{/}{nl}Arrow is split to pieces and bounces off attacking nearby enemies.
 SKILL_20150414_003766	{#DD5500}{ol}[Physical] - [Pierce]{/}{/}{nl}Run backwards and shoot multiple arrows forward.
 SKILL_20150414_003767	{memo X}Make a spiral arrow that penetrates the enemy. The shot gives 1 additional attack.
-SKILL_20150414_003768	Create standing shield to block enemy and enemy's range attack.
-SKILL_20150414_003769	Collect stones bullets to use for Stone Shot. Cannot be obtained in towns.
+SKILL_20150414_003768	Plant an upright shield to block hostile enemies as well as their ranged attacks.
```

Comment:
um, what about

'Plant an upright shield to block hostile enemies and their ranged attacks.'